### PR TITLE
Add template styles

### PR DESCRIPTION
**Pull request by vegeorgi**
http://ux.telerik.com/wireframes/NS%20Starter%20Kit/start.html#g=1&p=tab_strip

### CODE REVIEW

File: tabs/tabs-view-model.ts [hunk 1]
```diff
@@ -18,4 +19,8 @@ export class TabsViewModel extends Observable {
             this.notifyPropertyChange("title", value);
         }
     }
-}
+
+    get getIconPath(): string {
```
Comment (manoldonev) [inline tabs/tabs-view-model.ts]: I think a better name for the getter would be just iconPath not getIconPath.

File: tabs/tabs-view-model.ts [hunk 2]
```diff
@@ -18,4 +19,8 @@ export class TabsViewModel extends Observable {
...
+        return isAndroid ? "res://" : "res://tabIcons/";
+    }
+}
```
Comment (manoldonev) [inline tabs/tabs-view-model.ts]: There is a tslint error here: 
`ERROR: app/tabs/tabs-view-model.ts[26, 2]: file should end with a newline`

Make sure you execute `npm run lint` in a tns-created app after modifying ts/js files.

File: tabs/tabs-view-model.ts (hunk 1 above)
Comment (vegeorgi) [inline tabs/tabs-view-model.ts]: 👍

File: tabs/tabs-view-model.ts (hunk 2 above)
Comment (vegeorgi) [inline tabs/tabs-view-model.ts]: 👍

**Pull request merged**